tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -201,6 +203,8 @@ void UpdateStatsCollated(
     const std::unique_ptr<TensorDataSet>& data, const TensorInputTarget& target,
     int num_targets,
     const std::unordered_map<int32, std::vector<int>>& leaf_examples,
+    const std::unordered_map<int32, std::vector<int>>&
+        leaf_feature_to_use_per_split,
```

Comment:
i think leaf and split are mutually exclusive. features_per_split (because essentially u want to draw features that will be used in each split). U can also add a comment if u think it is not descriptive enough